tests/sentry/integrations/github/test_issues.py

```diff
@@ -37,30 +40,37 @@ def setUp(self):
             self.model.add_organization(self.organization, self.user)
         self.integration = GitHubIntegration(self.model, self.organization.id)
         self.min_ago = iso_format(before_now(minutes=1))
+        self.repo = "getsentry/sentry"
 
-    def _check_proxying(self) -> None:
-        assert len(responses.calls) == 1
-        request = responses.calls[0].request
-        assert request.headers[PROXY_OI_HEADER] == str(self.integration.org_integration.id)
-        assert request.headers[PROXY_BASE_URL_HEADER] == "https://api.github.com"
-        assert PROXY_SIGNATURE_HEADER in request.headers
+    @fixture(autouse=True)
+    def stub_get_jwt(self):
+        with patch.object(client, "get_jwt", return_value="jwt_token_1"):
+            yield
```

Comment:
I replaced `@patch("sentry.integrations.github.client.get_jwt", return_value="jwt_token_1")` with this automatic stub for a bunch of the tests